Sources/YCoreUI/Extensions/Foundation/String+Localizable.swift

```diff
@@ -12,7 +12,7 @@ public extension String {
     /// Gets a localized string resource using the string's current value as key
     /// - Parameter bundle: the bundle containing the localized string resource to use. Default = main app bundle.
```

Comment:
Need to update the doc comments to add the new parameter. You'll need to switch from `- Parameter` to `- Parameters` (temporarily cut the comments, and type Cmd+/ to have it regenerate the comments, paste the old comments somewhere else temporarily, copy over the old values, add in the field for tableName, then clean up).